.appveyor.yml

```diff
@@ -4,9 +4,10 @@
 
 environment:
   matrix:
+    - nodejs_version: '11'
     - nodejs_version: '10'
-    - nodejs_version: '9'
     - nodejs_version: '8'
+    - nodejs_version: '6'
```

Comment:
This package doesn't support Node 6. Minimum requirement is Node 8.6

Reply:
Well, then it should be made clear in the `engines` field. Otherwise people might run it on older versions.

Reply:
https://github.com/okonet/lint-staged/issues/515

Reply:
GitLab runner, SSH runner, tools like Surf / Deployer, ... which we use in CI and locally.

Reply:
Sorry I’m not following. I think the solution is okay since king-stages isn’t supposed to be used on CI so it shouldn’t break it. We have safety nets built in for runtime.